src/notation/view/widgets/editstyle.cpp

```diff
@@ -39,12 +41,15 @@
 
 using namespace mu::notation;
 using namespace mu::ui;
+using namespace mu::framework;
 
 static const QChar GO_NEXT_ICON = iconCodeToChar(IconCode::Code::ARROW_RIGHT);
 static const QChar GO_PREV_ICON = iconCodeToChar(IconCode::Code::ARROW_LEFT);
 static const QChar OPEN_FILE_ICON = iconCodeToChar(IconCode::Code::OPEN_FILE);
 static const QChar RESET_ICON = iconCodeToChar(IconCode::Code::REDO);
 
+static const Settings::Key STYLE_MENU_ORDER("notation","ui/style_menu_order");
```

Comment:
I believe we're using camelCase (rather dromedarCase) here, so `ui/styleMenuOrder`

Reply:
fixed